crates/rooch/src/commands/event.rs

```diff
@@ -29,10 +30,14 @@ pub enum EventSubCommand {
 
 #[derive(Debug, clap::Parser)]
 pub struct GetEventsByEventHandle {
+    /// Struct name as `<ADDRESS>::<MODULE_ID>::<STRUCT_NAME><TypeParam1?, TypeParam2?>`
+    /// Example: `0x123::event_test::WithdrawEvent --cursor 0 --limit 1`
     #[clap(long = "event_handle_type")]
     event_handle_type: StructTag,
+    /// start position
     #[clap(long)]
     cursor: Option<u64>,
+    /// end position
```

Comment:
In general, `end position` is equal to `cursor + limit`.
change `end position` to ` Max number of items returned per page`